src/shogun/lib/parameter_observers/ObservedValue.h

```diff
@@ -81,6 +82,8 @@ namespace shogun
 		int64_t m_step;
 		/** Parameter's name */
 		std::string m_name;
+        /** Untyped value */
+        Any m_any_value;
```

Comment:
@karlnapf This way we will store directly also the `Any` reference (I don't know why the formatting is messed up :/).

Reply:
Store value or reference?

Reply:
I would say by value because it is more self-contained.